src/SELF_DGModel2D_t.f90

```diff
@@ -461,15 +458,15 @@ subroutine Read_DGModel2D_t(this,fileName)
     integer :: firstElem
     integer :: N
 
-    if(this%decomp%mpiEnabled) then
+    if(this%mesh%decomp%mpiEnabled) then
       call Open_HDF5(fileName,H5F_ACC_RDWR_F,fileId, &
-                     this%decomp%mpiComm)
+                     this%mesh%decomp%mpiComm)
```

Comment:
Same as with the parallel writes, we need tests that cover parallel read of the pickup files. I'll add a single domain and multi-domain advection-diffusion 2d tests that load pickup files from one of the other advection-diffusion-2d runs

Reply:
resolved with [4f7a7a0](https://github.com/FluidNumerics/SELF/pull/53/commits/4f7a7a0e157b85dc677092583d328816442ade81)